src/xma/src/xmaapi/xmaxclbin.cpp

```diff
@@ -35,27 +36,32 @@ static int get_xclbin_iplayout(char *buffer, XmaXclbinInfo *xclbin_info);
 static int get_xclbin_mem_topology(char *buffer, XmaXclbinInfo *xclbin_info);
 static int get_xclbin_connectivity(char *buffer, XmaXclbinInfo *xclbin_info);
```

Comment:
const char* buffer

Reply:
this change is coming in separate pr for these other functions